platform/darwin/src/MGLStyleLayer.mm.ejs

```diff
@@ -229,6 +230,19 @@ namespace mbgl {
 <% if (paintProperties.length) { -%>
 #pragma mark - Accessing the Paint Attributes
 
++ (NSArray *)transitionKeys
+{
+    NSMutableArray *transitionKeys = [NSMutableArray array];
+<% for (const property of paintProperties) { -%>
+    [transitionKeys addObject:@"<%- camelizeWithLeadingLowercase(property.name) %>"];
+<% if (property.original) { -%>
+    [transitionKeys addObject:@"<%- camelizeWithLeadingLowercase(originalPropertyName(property)) %>"];
+<% } -%>
+<% } -%>
+
+    return transitionKeys;
+}
+
```

Comment:
@incanus I was using master but retarget to rebase release-ios-v3.5.0-android-v5.0.0